cnf-certification-test/operator/operator.go

```diff
@@ -0,0 +1 @@
+package operator
```

Comment:
Should this file be deleted?

Reply:
Yeah, looks like new file. I think I can add some functions inside provider/operators.go